.vib/owncloud/goss/goss.yaml

```diff
@@ -1,18 +1,15 @@
+http:
+  https://owncloud:{{ .Vars.service.ports.https }}:
+    status: 400
+    allow-insecure: true
 command:
-  user-id-test:
-    exec: if [ "$(id -u)" -eq 0 ]; then exit 1; fi
+  check-user-info:
+    exec: id
     exit-status: 0
-    stdout: []
-    stderr: []
+    stdout:
+      - uid={{ .Vars.containerSecurityContext.runAsUser }}
+      - /groups=.*{{ .Vars.podSecurityContext.fsGroup }}/
 file:
-  /opt/bitnami/apache/conf/extra/httpd-manual.conf:
-    mode: "0664"
-    filetype: file
-    exists: true
-  /opt/bitnami/mysql:
-    mode: "0755"
-    filetype: directory
-    exists: true
   /bitnami/owncloud/data:
```

Comment:
The correct path to be used would be `/bitnami/owncloud` as that is the one used in the volumeMount:

https://github.com/bitnami/charts/blob/8ec7c240ad509a21d938e1627b7281a088c54a3e/bitnami/owncloud/templates/deployment.yaml#L229